selfdrive/car/hyundai/values.py

```diff
@@ -275,7 +275,10 @@ class Buttons:
       b'\xf1\x81640J0051\x00\x00\x00\x00\x00\x00\x00\x00',
       b'\xf1\x81640K0051\x00\x00\x00\x00\x00\x00\x00\x00',
     ],
-    (Ecu.eps, 0x7d4, None): [b'\xf1\x00LX2 MDPS C 1.00 1.03 56310-S8020 4LXDC103',],
+    (Ecu.eps, 0x7d4, None): [
+      b'\xf1\x00LX2 MDPS C 1,00 1,03 56310-S8020 4LXDC103',
```

Comment:
can we add a comment next to it?

Reply:
before I opened this PR, I looked at honda to see if there was anything like that, and there is not, I will add it to both I guess